### DISCUSSION TITLE
route path parameter

### DISCUSSION THREAD

**Discussion by xingdong123** [apache/apisix#7763] [General]
route路径参数支持吗   
/aa/{id}/bb  像这种，我测试这种路径不成功，
是不是不支持这种，或者有其它替代方案吗？

**Comment by tzssangglass** [comment]
use `radixtree_uri_with_parameter `, ref: https://github.com/apache/apisix/blob/master/docs/zh/latest/router-radixtree.md#5-%E5%8F%82%E6%95%B0%E5%8C%B9%E9%85%8D
